udata_front/theme/gouvfr/templates/reuse/display.html

```diff
@@ -98,7 +98,7 @@ <h1 class="fr-mb-0 fr-h1">
         </div>
     </section>
 
-    <section class="fr-grid-row fr-grid-row--gutters fr-mt-4w fr-mb-6w container">
+    <section class="fr-grid-row fr-grid-row--gutters fr-mt-4w fr-mb-6w fr-container">
```

Comment:
Woops, I have an issue with the reuse tabs container here. See:
![image](https://user-images.githubusercontent.com/28541902/172415300-07d5b2c6-ca7e-4902-b5db-c2e8eb769695.png)

Reply:
Done ! This was the only one with a different markup